### PR TITLE
Add basic documentation for native histograms

### PR DESCRIPTION
**Pull request by krajorama**
#### What this PR does

see title

#### Which issue(s) this PR fixes or relates to

Fixes: #3478 

#### Checklist

- N/A Tests updated
- [x] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: docs/sources/mimir/configure/configure-native-histograms-ingestion.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+weight: 160
+---
+
+# Configure native histograms
+
+To enable support for ingesting Prometheus native histograms over the [remote write API]({{< relref "../references/http-api/#remote-write">}}) endpoint, set the flag `-ingester.native-histograms-ingestion-enabled=true` on ingesters.
+
+To enable support for querying native histograms together with [Grafana Mimir query sharding]({{< relref "../operators-guide/architecture/query-sharding">}}), set the flag `-query-frontend.query-result-response-format=protobuf` on query frontends.
+
+**Note:** native histograms is an experimental feature of Grafana Mimir and of [Prometheus](https://prometheus.io/docs/prometheus/latest/feature_flags/#native-histograms).
```
Comment (osg-grafana) [inline docs/sources/mimir/configure/configure-native-histograms-ingestion.md:14]:
```suggestion
> **Note:** Native histograms is an experimental feature of Grafana Mimir and of [Prometheus](https://prometheus.io/docs/prometheus/latest/feature_flags/#native-histograms).
```

**Review by osg-grafana — APPROVED** [review]
Unblocking with a nit

File: docs/sources/mimir/configure/configure-native-histograms-ingestion.md (hunk 1 above)
Comment (krajorama) [reply docs/sources/mimir/configure/configure-native-histograms-ingestion.md]: fixed

File: CHANGELOG.md [hunk 2]
```diff
@@ -67,6 +67,8 @@
 
 ### Documentation
 
+* [FEATURE] Add instructions on how to configure native histograms. #4527
```
Comment (osg-grafana) [inline CHANGELOG.md:70]:
```suggestion
* [FEATURE] Add instructions about how to configure native histograms. #4527
```

**Review by osg-grafana — APPROVED** [review]
Ship it :)

**Pull request merged**